### DISCUSSION TITLE
vaultwarden push notifications

### DISCUSSION THREAD

**Discussion by navels** [tteck/Proxmox#3389] [Help Needed] [answered]
Anyone gotten this to work with the proxmox installation of vaultwarden: https://github.com/dani-garcia/vaultwarden/wiki/Enabling-Mobile-Client-push-notification

**Comment by Icki153** [comment, marked as answer]
Hi Navels,
tried it right now. Because i want also to get it to work. And it works like described in your link. Get the ID and Key and select the region.
Then change the file "/opt/vaultwarden/.env" and add like in the Guide the options. For me in EU region it looks like this.

```
PUSH_ENABLED=true
PUSH_INSTALLATION_ID=CHANGETOYOURID
PUSH_INSTALLATION_KEY=CHANGETOYOURKEY
PUSH_RELAY_URI=https://api.bitwarden.eu
PUSH_IDENTITY_URI=https://identity.bitwarden.eu
```
 
 Then save the file. Reboot the LXC. Then you can see in Admin Panel that Push is enabled and also your ID and Key.
 
To get it to work delete the data from App on Android or delete and reinstall the App on IOS. Then it asks after new login if you want to enable push.

Hope this helps.

**Reply by navels** [reply]
Thanks!

**Comment by ma3d1um** [comment]
Hi, I'm also curious about this, but I'm running the Alpine version and cannot find `/opt/vaultwarden/.env` there.
Does this mean that this only works with the "normal" LXC? Thanks!

**Reply by Icki153** [reply]
Yes this was only for the normal LXC. But take a look to file „ /etc/conf.d/vaultwarden“ maybe there you can configure the right options.

**Reply by ma3d1um** [reply]
Wow, great this works -- thanks!!! 
For others that would like this, the steps are:

1. Open the Vaultwarden Alpine shell (login with `root`, no password)
2. Type `nano /etc/conf.d/vaultwarden`
3. Remove the `#` on the above mentioned lines, so that it looks like this:

```
## Enables push notifications (requires key and id from https://bitwarden.com/host)
export PUSH_ENABLED=true
export PUSH_INSTALLATION_ID=<YOUR ID>
export PUSH_INSTALLATION_KEY=<YOUR KEY>
## Don't change this unless you know what you're doing.
export PUSH_RELAY_URI=https://api.bitwarden.eu
export PUSH_IDENTITY_URI=https://identity.bitwarden.eu
```
